### DISCUSSION TITLE
Form: 有时需要暂存一个没有item对应的值，怎么做才能在validateFields时收集到它？

### DISCUSSION THREAD

**Discussion by Lpbzzz** [ant-design/ant-design#36629] [Q&A]
如题所示。有时业务上可能会在编辑时传一个code之类的字段，不对它做任何操作。不想额外写state存储，在validateFields时怎么收集到它呢？

**Comment by afc163** [comment]
initialValues 里面带一个，或者 validateFields 里加上。

**Reply by Lpbzzz** [reply]
用formily时直接赋个值就可以。不过这也不是什么很必要的需求..

**Comment by Lpbzzz** [comment]
刚试了一下：https://stackblitz.com/edit/react-cuxdww-codhsw?file=demo.js   initialValues里加个code，似乎并不能让onFinish收集到

**Reply by afc163** [reply]
可以加个 hidden 的 Form.Item

```
      <Form.Item label="code" name="code" hidden>
        <input />
      </Form.Item>
```
